lib/govuk_index/publishing_event_worker.rb

```diff
@@ -11,24 +11,16 @@ class MissingBasePath < StandardError; end
   class PublishingEventWorker < Indexer::BaseWorker
     notify_of_failures
 
-    def perform(routing_key, payload)
+    def perform(messages)
       actions = ElasticsearchProcessor.new
```

Comment:
`actions` seems like a weird name here. Can we call it processor instead?

Reply:
done